google-cloud-text_to_speech/Gemfile

```diff
@@ -0,0 +1,12 @@
+source "https://rubygems.org"
+
+gemspec
+
+gem "gcloud-jsondoc",
+    git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
+    branch: "gcloud-jsondoc"
+
```

Comment:
This is needed to get the build passing for Ruby 2.0:

```ruby
if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.1")
  # WORKAROUND: builds are failing on Ruby 2.0.
  # We think this is because of a bug in Bundler 1.6.
  # Specify a viable version to allow the build to succeed.
  gem "jwt", "~> 1.5"
  gem "kramdown", "< 1.17.0" # Error in yard with 1.17.0
end
```

Reply:
Added.